deployment/chainloop/values.yaml

```diff
@@ -182,6 +187,14 @@ controlplane:
       clientID: ""
       clientSecret: ""
 
+    ## @param controlplane.auth.allowList.rules List of domains or emails to allow
+    ## @param controlplane.auth.allowList.selectedRoutes List of selected routes to allow
+    ## @param controlplane.auth.allowList.customMessage Custom message to display when a user is not allowed
+    allowList:
```

Comment:
if we do not want to render this by default we should comment this out and set instead of @param, @extra

Reply:
same about onboarding

Reply:
I didn't know that was an option...I'll do it.